front/playwright.config.ts

```diff
@@ -63,5 +63,15 @@ export default defineConfig({
       },
       dependencies: ['setup'],
     },
+    {
+      name: 'webkit',
+      use: {
+        browserName: 'webkit',
+        launchOptions: {
+          slowMo: 500, // Slows down WebKit interactions by 500 milliseconds
+        },
+      },
+      dependencies: ['setup'],
+    },
```

Comment:
shouldn't we only use test.slow() for specifics tests? I'm afraid that the actual delay could hide some performance issues

Reply:
WebKit is around 1.7 times slower than Firefox and Chromium, according to feedback from the Playwright community. Therefore, adding `slowMo` ( only 500ms) limits the number of tests on which we apply `test.slow()`.